packages/cli/src/cmd/run.ts

```diff
@@ -37,17 +37,16 @@ interface RunArguments extends Arguments {
 function setupDelayOverrides(args: RunArguments, testSettingOverrides: TestSettings) {
 	if (testSettingOverrides == null) testSettingOverrides = {}
 
-	if (args.fastForward ?? false) {
-		testSettingOverrides.stepDelay = args.fastForward
-		testSettingOverrides.actionDelay = args.fastForward
-	} else if (args.slowMo ?? false) {
-		testSettingOverrides.stepDelay = args.slowMo ?? testSettingOverrides.stepDelay
-		testSettingOverrides.actionDelay = args.slowMo ?? testSettingOverrides.actionDelay
-	}
-
-	testSettingOverrides.actionDelay = args.actionDelay ?? testSettingOverrides.actionDelay
-	testSettingOverrides.actionDelay = args.stepDelay ?? testSettingOverrides.stepDelay
+	testSettingOverrides.actionDelay = args.actionDelay ?? 0
+	testSettingOverrides.stepDelay = args.stepDelay ?? 0
```

Comment:
I think we should handle the negative case, let say if the user input value -1?
```suggestion
	testSettingOverrides.actionDelay = args.actionDelay && args.actionDelay > 0 ? args.actionDelay : 0
	testSettingOverrides.stepDelay = args.stepDelay && args.stepDelay > 0 ? args.stepDelay : 0
```